modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/GetEntryRenderDataMVCResourceCommand.java

```diff
@@ -522,6 +534,33 @@ else if (ctEntry.getChangeType() !=
 			}
 		}
 
+		String workflowView = null;
+
+		Map<String, String> workflowData = new LinkedHashMap<>();
+
+		if (rightModel instanceof WorkflowedModel) {
+			long groupId = 0;
+
+			if (rightModel instanceof GroupedModel) {
+				GroupedModel groupedModel = (GroupedModel)rightModel;
+
+				groupId = groupedModel.getGroupId();
+			}
+
+			if ((groupId != 0) &&
```

Comment:
No need to check for the groupId here. It's ok to pass in 0 for groupId for WorkflowedModels that are not GroupedModel. 

Non-grouped models that support workflow can be enabled in Applications Menu > Process Builder.